docs/proposals/autoscaling.md

```diff
@@ -152,27 +133,26 @@ For this proposal, the auto-scaler is a resource:
     }     
      
      
-     //abstracts the data analysis from the auto-scaler
+     //AutoScaleThresholdInterface abstracts the data analysis from the auto-scaler
      //example: scale by 1 (Increment) when RequestsPerSecond (Type) pass comparison (Comparison) of 50 (Value) for 30 seconds (Duration)
      type AutoScaleThresholdInterface interface {
         //called by the auto-scaler to determine if this threshold is met or not
         ShouldScale() boolean
      }
       
-     type StatisticType string
         
-     //generic type definition
+     //AutoScaleThreshold is a single statistic used to drive the auto-scaler in scaling decisions
      type AutoScaleThreshold struct {
-         //scale up or down by this increment
+         //Increment determines how the auot-scaler should scale up or down (positive number to scale up based on this threshold
+         //negative number to scale down by this threshold)
          Increment int
-         //by querying this statistic
-         //example: RequestsPerSecond StatisticType = "requestPerSecond"
-         Type StatisticType
-         //after this duration
+         //Selector is the statistics selector that determines how the Threshold receives data from aggregated (or single) statistics. 
+         Selector map[string]string        
+         //Duration is the time laps after which this threshold is considered passed
          Duration time.Duration
-         //when this value is passed
+         //Value is the number at which, after the duration is passed, this threshold is considered to be triggered
```

Comment:
To be more specific in the answer, I don't think that we care what the units are and that we leave it up to the statistics gather mechanism to publish how it stores stats (ms vs seconds, etc) and up to the user to define the correct value that matches the units.